src/youtubeMusic.ts

```diff
@@ -114,6 +116,20 @@ export default class YouTubeMusic {
       },
       { id: "skip", title: "Next Song", text: "$(chevron-right)" },
       { id: "cycleRepeat", title: "Not Repeating", text: "$(sync)" },
+      {
+        id: "thumbsdown",
+        title: "Thumbs Down",
+        text: "$(mui-thumbs-down)",
+        dynamicText: (isThumbsDown: boolean) =>
+        isThumbsDown ? "$(mui-thumbs-down-solid)" : "$(mui-thumbs-down)",
```

Comment:
Could we stick to consistent whitespace here (matching `thumbsup` below)
```suggestion
        dynamicText: (isThumbsDown: boolean) =>
          isThumbsDown ? "$(mui-thumbs-down-solid)" : "$(mui-thumbs-down)",
```